osmosis-1/osmosis.zone_assets.json

```diff
@@ -3439,17 +3439,24 @@
       "_comment": "BackBone Labs Liquid Staked OSMO $bOSMO"
     },
     {
-      "chain_name": "pundix",
-      "base_denom": "ibc/55367B7B6572631B78A93C66EF9FDFCE87CDE372CC4ED7848DA78C1EB1DCDD78",
-      "path": "transfer/channel-12618/transfer/channel-0/eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
+      "chain_name": "fxcore",
+      "base_denom": "ibc/46D8D1A6E2A80ECCB7CA6663086A2E749C508B68DA56A077CD26E6F4F9691EEE",
+      "path": "transfer/channel-2716/eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
+      "transfer_methods": [
+        {
+          "name": "Starscan",
+          "type": "external_interface",
+          "deposit_url": "https://starscan.io/fxbridge?from=fxcore&to=osmosis&token=PUNDIX"
+        }
+      ],
       "canonical": {
         "chain_name": "ethereum",
         "base_denom": "0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38"
       },
+      "osmosis_verified": true,
```

Comment:
Can I ask that we start with updating this PR to keep verified as 'false' so we can validate these changes first?
Then next PR we can set to 'true'.
The osmosis_unlisted property can be removed in either PR, just replace it with a listed date when we do so.

Reply:
ok